geoportailv3/static/js/catalog/layertreenode.html

```diff
@@ -1,14 +1,14 @@
 <a data-toggle="collapse" data-parent="#catalog-{{::layertreenodeCtrl.parentUid}}"
    href="#catalog-{{::layertreenodeCtrl.uid}}"
    ng-if="::layertreenodeCtrl.node.children && layertreenodeCtrl.depth < 3">
-  {{::layertreenodeCtrl.node.name}}
+  {{::layertreenodeCtrl.node.name | translate}}
```

Comment:
It's probably me not understanding when the filter rather than the directive should be used.